website/docs/plugin/which-sdk.mdx

```diff
@@ -41,6 +41,8 @@ should consider using the framework if you would benefit from the ability to:
 * Use nested attributes.
 * Use any type as the elements of a map.
 * Tell when an optional and computed field has been removed from a config.
+* Implement schema-based validation that is relative to the current attribute or applies to any list/map/set element.
```

Comment:
Suggest rewording "list/map/set to "list, map, or set"

Reply:
Sounds good, will adjust to the below (using GitHub's code suggestion feature)

```suggestion
* Implement schema-based validation that is relative to the current attribute or applies to any list, map, or set element.
```